src/frontend/src/pages/HomePage/components/TeamWorkPackageDisplay.tsx

```diff
@@ -0,0 +1,78 @@
+import WorkPackageCard from './WorkPackageCard';
+import PageBlock from '../../../layouts/PageBlock';
+import { wbsPipe } from '../../../utils/pipes';
+import Box from '@mui/material/Box';
+import { useAllTeams } from '../../../hooks/teams.hooks';
+import { useCurrentUser } from '../../../hooks/users.hooks';
+import LoadingIndicator from '../../../components/LoadingIndicator';
+import ErrorPage from '../../ErrorPage';
+import { useTheme } from '@mui/material';
+
+const TeamWorkPackageDisplay: React.FC = () => {
+  const theme = useTheme();
+  const user = useCurrentUser();
+  const { isLoading, isError, data: teams, error } = useAllTeams();
+  const myTeams = teams?.filter((team) => {
```

Comment:
you should check if teams have have been retrieved before calling filter on teams